.github/scripts/setup-manager-self-hosted.py

```diff
@@ -21,14 +36,11 @@ def initialize_manager_hosted():
 
     # Catch any exception that occurs so that we can gracefully teardown
     try:
-        # wait until machine launch is complete
-        with cd(manager_home_dir):
-            with settings(warn_only=True):
-                rc = run("timeout 20m grep -q '.*machine launch script complete.*' <(tail -f machine-launchstatus)").return_code
-                if rc != 0:
-                    run("cat machine-launchstatus.log")
-                    raise Exception("machine-launch-script.sh failed to run")
-
+        with settings(warn_only=True):
```

Comment:
I assume these are going to be removed once you are done testing things?

Reply:
Yes.  These will get removed.